.github/workflows/linux-meson-builds.yml

```diff
@@ -0,0 +1,43 @@
+name: Linux builds (basic) using meson build system
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: meson ${{matrix.cxx}}, C++${{matrix.std}}, ${{matrix.build_type}}
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        cxx:
+          - g++-11
+          - clang++-11
+        build_type: [debug, release]
+        std: [14, 17]
+        include:
+          - cxx: clang++-11
+            other_pkgs: clang-11
+
```

Comment:
I reduced the build matrix whilst I was trying to make this workflow work. Would you like me to try widening it to more compiler versions now it seems to be working?

Reply:
Nah, this is fine.